pyoozie/model.py

```diff
@@ -377,51 +361,25 @@ class Status(_OozieArtifact.StatusEnum):
         WAITING = _status(11)
 
     def __init__(self, *args, **kwargs):
-        # Phony declarations to appease pylint
-        self.actionNumber = None
-        self.consoleUrl = None
-        self.coordJobId = None
-        self.createdConf = None
-        self.createdTime = None
-        self.errorCode = None
-        self.errorMessage = None
-        self.externalId = None
-        self.externalStatus = None
-        self.id = None
-        self.lastModifiedTime = None
-        self.missingDependencies = None
-        self.nominalTime = None
-        self.pushMissingDependencies = None
-        self.runConf = None
-        self.status = None
-        self.toString = None
-        self.trackerUri = None
-        self.type = None
-        self._parent = None
+        self.status = CoordinatorAction.Status.UNKNOWN
+        super(CoordinatorAction, self).__init__(*args, **kwargs)
         self._workflow = None
-        super(CoordinatorAction, self).__init__(self, *args, **kwargs)
 
     def _validate_degenerate_fields(self):
         # For any fields that must be in sync, ensure they are.
         # If values are missing, extrapolate them
         coord_id, action = parse_coordinator_id(self.id)
-        if self.coordJobId:
-            if self.coordJobId != coord_id:
-                raise OozieException.parse_error("coordJobId does not match coordinator action ID")
-        else:
-            self.coordJobId = coord_id
-        if self.actionNumber:
-            if str(self.actionNumber) != str(action):
-                raise OozieException.parse_error("actionNumber does not match coordinator action ID")
-        else:
-            self.actionNumber = action
-        if self.toString:
-            if self.id not in self.toString:
-                raise OozieException.parse_error("toString does not contain coordinator action ID")
-            if not self.status.is_unknown() and str(self.status) not in self.toString:
-                raise OozieException.parse_error("toString does not contain status")
-        else:
-            self.toString = 'CoordinatorAction name[{}] status[{}]'.format(self.id, self.status)
+        self.coordJobId = self.coordJobId or coord_id
```

Comment:
Code here is logically the same as above, but rewritten so the linter won't be confused by the dynamic initialization that occurs in the `__init__` method.